models/wrf/tutorial/template/input.nml.template

```diff
@@ -130,7 +130,7 @@
                              'QNRAIN','QTY_RAIN_NUMBER_CONCENTR','TYPE_QNRAIN','UPDATE','999',
                              'U10','QTY_U_WIND_COMPONENT','TYPE_U10','UPDATE','999',
                              'V10','QTY_V_WIND_COMPONENT','TYPE_V10','UPDATE','999',
-                             'T2','QTY_TEMPERATURE','TYPE_T2','UPDATE','999',
+                             'T2','QTY_2M_2M_TEMPERATURE','TYPE_T2','UPDATE','999',
```

Comment:
you've got 2 2M in here:

QTY_2M_**2M**_TEMPERATURE

Reply:
done